msal/src/Microsoft.Identity.Client/MigrationAid/MigrationAid.cs

```diff
@@ -10,20 +11,23 @@ namespace Microsoft.Identity.Client
     /// In MSAL.NET 1.x, was representing a User. From MSAL 2.x use <see cref="IAccount"/> which represents an account
     /// (a user has several accounts). See https://aka.ms/msal-net-2-released for more details.
     /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
this is to avoid the type (or members) to be presented by IntelliSense, as we don't want developers to use them in MSAL 2.x. They are only here to help them migrate from MSAL 1.x